apps/schools/management/commands/populateOLP.py

```diff
@@ -0,0 +1,258 @@
+import os
```

Comment:
This script directly imports the schools and creates the boundaries on dubdubdub instead of coming via the EMS route. It throws the db out of sync and probably this is not the way we want to do it going forward. However, in the `unified db`, anyway the `Institution` table will be common and hence, the sync will remain despite which platform we use to create new schools and boundaries.